### DISCUSSION TITLE
Hello dev team!  Can I know how to use this library in Laravel framework? Version 10

### DISCUSSION THREAD

**Discussion by awesome1128** [GrapesJS/grapesjs#5301] [Q&A]
Nice to meet you.
I found you today, oh you did great job!
Since yesterday, I have started a website development using Laravel v10.
I want to use your library in my project because I have non-technical guys.
Can I know how to use your library in laravel?
Please let me know, thanks.

**Comment by artf** [comment]
Maybe [this one](https://github.com/mjawad096/laravel-grapesjs) can help
